homeassistant/components/blebox/button.py

```diff
@@ -0,0 +1,54 @@
+"""BleBox button entities implementation."""
+from __future__ import annotations
+
+from homeassistant.components.button import ButtonDeviceClass, ButtonEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import BleBoxEntity, create_blebox_entities
+from .const import BLEBOX_TO_HASS_DEVICE_CLASSES
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up a BleBox button entry."""
+    create_blebox_entities(
+        hass, config_entry, async_add_entities, BleBoxButtonEntity, "buttons"
+    )
+
+
+class BleBoxButtonEntity(BleBoxEntity, ButtonEntity):
+    """Representation of BleBox buttons."""
+
+    def __init__(self, feature):
+        """Initialize a BleBox button feature."""
+        super().__init__(feature)
+        self._attr_device_class = BLEBOX_TO_HASS_DEVICE_CLASSES["button"]
+        self._attr_icon = self.get_icon()
+
+    @property
+    def device_class(self) -> ButtonDeviceClass | None:
```

Comment:
Since you are using `self._attr_device_class` instance attribute I think you can drop the property.